docs/source/llms/transformers/guide/index.rst

```diff
@@ -685,105 +685,6 @@ This input format requires that both the bitrate has been set prior to conversio
     audio transformers pipelines being set as expecting ``binary`` (``bytes``) data. The serving endpoint cannot accept a union of types, so a particular model instance must choose one
     or the other as an allowed input type.
 
-.. _transformers-save-pretrained-guide:
-
-Storage-Efficient Model Logging with ``save_pretrained`` Option
```

Comment:
note: Moving to a new dedicated page to clearly illustrate the difference between multiple saving methods.

Reply:
great call